console/src/main/java/org/eclipse/rdf4j/console/util/UncloseableOutputStream.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.console.util;
+
+import java.io.FilterOutputStream;
+import java.io.OutputStream;
+
+/**
+ * Uncloseable output stream, useful for keeping console output open
```

Comment:
Ahh. I see what you are doing here. Can you configure it to close on exit?

Reply:
The (filtered/wrapped) outputstream should be closed by the JLine terminal.

I'd suggest to move this class to `org.eclipse.rdf4j.common.io`, since that package already contains an `UncloseableInputstream` (with a `doClose` method to force closing the stream)